google/resource_compute_address_test_configs.go

```diff
@@ -0,0 +1,52 @@
+// ----------------------------------------------------------------------------
```

Comment:
Nothing wrong with this, but why separate out the configs into their own file?  Normally we just put 'em at the bottom of the main file.